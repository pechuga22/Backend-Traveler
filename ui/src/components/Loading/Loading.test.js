import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { render } from '@testing-library/react'

import Loading from './Loading'

describe('./components/Loading/Loading.js', () => {
  describe('.render()', () => {
    it('renders text Loading', () => {
      const { getByText } = render(<Loading />)
      const loadingText = getByText(/loading/i)
    })    

    it('should render Loading component', () => {
      const component = shallow(<Loading />)
      expect(toJson(component)).toMatchSnapshot()
    })
  })
})
