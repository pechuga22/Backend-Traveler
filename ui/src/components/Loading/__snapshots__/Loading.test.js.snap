// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./components/Loading/Loading.js .render() should render Loading component 1`] = `
<Dimmer
  active={true}
  inverted={true}
>
  <Loader
    size="large"
  >
    Loading...
  </Loader>
</Dimmer>
`;
